test/integration/file-uploads/request-body.test.js

```diff
@@ -645,4 +650,65 @@ describe('file upload in request body', function () {
                 .to.equal('Binary file load error: file resolver interface mismatch');
         });
     });
+
+    (IS_BROWSER ? describe.skip : describe)('should upload large files correctly', function () {
+        var testUploadFile = 'test/fixtures/upload-file-large.json';
```

Comment:
Nitpick: We follow a convention of naming a constant variable in all caps.
```suggestion
        var TEST_UPLOAD_FILE = 'test/fixtures/upload-file-large.json';
```